fili-core/src/main/java/com/yahoo/bard/webservice/table/resolver/GranularityComparator.java

```diff
@@ -15,6 +15,24 @@
 public class GranularityComparator implements Comparator<PhysicalTable> {
 
     private static final Logger LOG = LoggerFactory.getLogger(GranularityComparator.class);
+    private static GranularityComparator granularityComparator;
```

Comment:
Make this a `private static final` and call the constructor to assign it statically.